src/style.css

```diff
@@ -1 +1,84 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Heavy.ttf');
+  font-weight: bold;
+}
+
+body {
+  padding: 0;
+}
+
+.search-bar {
+  margin: 20px 0;
+}
+
+.search-bar__input {
+  box-sizing: border-box;
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  font-family: Avenir, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. Add a generic font family as a fallback after the custom font-family like `sans-serif`.